### DISCUSSION TITLE
Can you customize the table view in the dashboard?

### DISCUSSION THREAD

**Discussion by gisete** [tinacms/tinacms#4097] [Help]
I'm trying to see if Tina is the right tool for my project and I couldn't find a clear answer for this.

Is it possible to add what fields are shown in the dashboard? For example, I don't care for showing the file name or extension here and would add other items instead.
![image](https://github.com/tinacms/tinacms/assets/10730651/b8b4d8d0-3481-4dea-81e1-d91a53bc261a)

**Comment by muffn** [comment]
I would like to know the same, did you figure anything out @gisete?

**Comment by kldavis4** [comment]
This is not currently customizable
